lib/mandrill_dm/message.rb

```diff
@@ -248,6 +258,10 @@ def return_string_value(field)
       mail[field] ? mail[field].to_s : nil
     end
 
+    def return_time_parsed_string_value(field)
+      mail[field] ? Time.parse(mail[field].value).utc.strftime("%Y-%m-%d %H:%M:%S") : nil
```

Comment:
@kshnurov as far as I can tell, `Mail:OptionalField` does not implement a `get_value` method. `value` is available but returns a string. I'm not doing this work flippantly.

So look I'd really like to get this merged, but its seeming like an uphill battle here. If this isn't close to ready in your opinion then I may just keep this as my own fork for my own uses.

What else can I do? Either your library won't support a (really nice) Mandrill feature, or you have to make some concessions around this work. 

Ideally I'd like to see the gem require passing "Date-y" objects as send_at (not time strings), and an error should be raised if you try to provide anything that isn't a date object. Would that be more acceptable to you?